src/services/rag/components/retrievers/lightrag.py

```
@@ -0,0 +1,136 @@
+        try:
+            from lightrag import LightRAG, QueryParam
+            from lightrag.llm.openai import openai_complete_if_cache
+
+            from src.services.embedding import get_embedding_client
+            from src.services.llm import get_llm_client
+
+            llm_client = get_llm_client()
+            embed_client = get_embedding_client()
+
+            # LLM function using services (SYNC - LightRAG expects sync functions)
+            def llm_model_func(prompt, system_prompt=None, history_messages=[], **kwargs):
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)